src/components/appRouter.js

```diff
@@ -330,8 +330,8 @@ define(['loading', 'globalize', 'events', 'viewManager', 'skinManager', 'backdro
         }
 
         if (shouldExitApp) {
-            if (appHost.supports('exit')) {
-                appHost.exit();
+            if (appHost.default.supports('exit')) {
+                appHost.default.exit();
```

Comment:
How about these? https://github.com/jellyfin/jellyfin-web/blob/37abd530eb5b462d8e5493d9d7e48da08d661ecb/src/components/appRouter.js#L286-L287
ES6 import hack at the beginning would be better, imo